src/main/resources/META-INF/native-image/software.amazon.awssdk/crt/aws-crt/reflect-config.json

```diff
@@ -0,0 +1,60 @@
+[
+  {
+    "name":"java.lang.Object",
+    "allDeclaredFields":true,
+    "queryAllDeclaredMethods":true
+  },
+  {
+    "name":"java.lang.Throwable",
+    "methods":[{"name":"getSuppressed","parameterTypes":[] }]
+  },
+  {
+    "name":"java.util.concurrent.ForkJoinTask",
```

Comment:
Those are files that have been generated by running the Unit-Tests and some of them seem to be related to test cases only. Those should be not needed as the native-image tests require a normal test run in advance so that those files are automatically created. I cleaned up the obvious ones and will push an update.